README.md

```diff
@@ -37,26 +22,29 @@ https://dev.futurice.com:8081/vimma/
 
 ## Running on your machine:
 
-Create a Python3 virtual enviroment and use it for most commands.
```

Comment:
Agreed – I haven't been running these by hand for a long time. Just trying to future-proof for the day when our convenience helpers won't work and no one remembers these commands :)